wlm_server/user/admin.py

```diff
@@ -3,4 +3,31 @@
 
 from .models import User
 
-admin.site.register(User, UserAdmin)
+@admin.register(User)
+class CustomUserAdmin(UserAdmin):
```

Comment:
How about setting `last_login` field as read-only?

See also: [`readonly_fields`](https://docs.djangoproject.com/en/5.1/ref/contrib/admin/#django.contrib.admin.ModelAdmin.readonly_fields)

Reply:
See also: https://forum.djangoproject.com/t/how-to-set-last-login-to-read-only-in-the-admin-pages/29213

Reply:
It looks much better👍